website/content/docs/connect/config-entries/service-splitter.mdx

```diff
@@ -1,54 +1,567 @@
+
+## Configuration model
+
+The following list outlines field hierarchy, language-specific data types, and requirements in a service splitter configuration entry. Click on a property name to view details about the configuration.
+
+<Tabs>
+
+<Tab heading="HCL and JSON" group="hcl">
+
+- [`Kind`](#kind): string | required
+- [`Name`](#name): string | required
+- [`Namespace`](#namespace): string <EnterpriseAlert inline />
+- [`Partition`](#partition): string <EnterpriseAlert inline />
+- [`Meta`](#meta): map 
+- [`Splits`](#splits): map | required
+	- [`Weight`](#splits-weight): number | required
+	- [`Service`](#splits-service): string | required
+	- [`ServiceSubset`](#splits-servicesubset): string 
+  - [`Namespace`](#splits-namespace): string <EnterpriseAlert inline />
+  - [`Partition`](#splits-partition): string <EnterpriseAlert inline />
+  - [`RequestHeaders`](#splits-requestheaders): map
+    - [`Add`](#splits-requestheaders): map 
+    - [`Set`](#splits-requestheaders): map 
+    - [`Remove`](#splits-requestheaders): map 
+  - [`ResponseHeaders`](#splits-responseheaders): map 
+    - [`Add`](#splits-responseheaders): map 
+    - [`Set`](#splits-responseheaders): map 
+    - [`Remove`](#splits-responseheaders): map 
+
+</Tab>
+
+<Tab heading="YAML" group="yaml">
+
+- [`apiVersion`](#apiversion): string | required
+- [`kind`](#kind): string | required
+- [`metadata`](#metadata): object | required
+	- [`name`](#metadata-name): string | required
+	- [`namespace`](#metadata-namespace): string | optional <EnterpriseAlert inline />
+- [`spec`](#spec): object | required
+	- [`splits`](#spec-splits): list | required
+		- [`weight`](#spec-splits-weight): float32 | required
+		- [`service`](#spec-splits-service): string | required
+	  - [`serviceSubset`](#spec-splits-servicesubset): string
+    - [`namespace`](#spec-splits-namespace): string <EnterpriseAlert inline />
+    - [`partition`](#spec-splits-partition): string <EnterpriseAlert inline />
+    - [`requestHeaders`](#spec-splits-requestheaders): HTTPHeaderModifiers 
+      - [`add`](#spec-splits-requestheaders): map 
+      - [`set`](#spec-splits-requestheaders): map 
+      - [`remove`](#spec-splits-requestheaders): map 
+    - [`responseHeaders`](#spec-splits-responseheaders): HTTPHeaderModifiers 
+      - [`add`](#spec-splits-responseheaders): map 
+      - [`set`](#spec-splits-responseheaders): map 
+      - [`remove`](#spec-splits-responseheaders): map 
+
+</Tab>
+</Tabs>
+
+## Complete configuration
+
+When every field is defined, a service splitter configuration entry has the following form:
+
+<Tabs>
+
+<Tab heading="HCL" group="hcl">
+
+```hcl
+Kind      = "service-splitter"           ## string  | required
+Name      = "config-entry-name"          ## string  | required
+Namespace = "main"                       ## string
+Partition = "partition"                  ## string 
+Meta      = {                            ## map
+  key     = "value"            
+  }
```

Comment:
The formatting on these indented objects looks a little strange. I think it's because the `=` symbols are lined up. How do we usually format HCL/JSON?

I'd expect it to look more like `gofmt`'s formatting of struct fields:

```Go
foo := Foo{
	AAA:  "a",
	BBBB: "b",
	CCCCC: Other{
		D: "d",
	},
}
```

Note the scalar field values are lined up with each other, but not `CCCCC` or its fields.

Reply:
@boxofrad Just to be clear, you're saying that we should line  values up, not the symbols?, so:

```
Kind =      "service-splitter"
Name =      "config-entry-name"
Namespace = "main"                   
Partition = "partition"  
```

Reply:
Sorry, my bad for not being clear! Go with whatever the established style is, but I'd expect the above snippet to look more like this:

```hcl
Kind      = "service-splitter"
Name      = "config-entry-name"
Namespace = "main"
Partition = "partition"
Meta = {
  key       = "value"
  other_key = "value"            
}
```

Actually, maybe run [`hclfmt`](https://github.com/fatih/hclfmt) on it?